lib/get-compiler-options.js

```diff
@@ -1,6 +1,6 @@
 const { dirname } = require('path');
+const { createParsedCommandLine, resolveVueCompilerOptions } = require('@vue/language-core');
```

Comment:
This was intentionally `require`'d within the `getVueCompilerOptions` because we don't want to require it unless Vue is involved.